src/algorithms.jl

```diff
@@ -125,6 +125,49 @@ function Lanczos(;
     return Lanczos(orth, krylovdim, maxiter, tol, eager, verbosity)
 end
 
+"""
+    BlockLanczos(; krylovdim=KrylovDefaults.blockkrylovdim[],
+            maxiter=KrylovDefaults.maxiter[],
+            tol=KrylovDefaults.tol[],
+            orth=KrylovDefaults.orth,
+            eager=false,
+            verbosity=KrylovDefaults.verbosity[],
+            qr_tol::Real=KrylovDefaults.tol[])
+
+The block version of [`Lanczos`](@ref) is suited for solving eigenvalue problems with repeated dominant eigenvalues.
```

Comment:
Dominant seems to refer to `:LM` (at least to me), whereas it would work for both `:LR` and `:SR` as well.

```suggestion
The block version of [`Lanczos`](@ref) is suited for solving eigenvalue problems with repeated extremal eigenvalues.
```